frontend/src/components/Room/RoomSidebarChat/Chat/Chat.jsx

```diff
@@ -42,6 +42,14 @@ const Chat = ({chatData, sendMessage, connections, localPeer}) => {
         setNewMessage(event.target.value)
     };
 
+    const handleKeyPress = (event) => {
+        if(event.key === 'Enter' && newMessage.length > 0) {
+            event.preventDefault()
```

Comment:
the send method below does exactly the same things as the 3 lines here. Always keep in mind: don't repeat yourself (DRY)